package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "gaia-button",
+  "version": "0.0.5",
+  "homepage": "https://github.com/gaia-components/gaia-button",
+  "license": "MIT",
+  "main": "gaia-button.js",
+  "devDependencies": {
```

Comment:
Ah you're right! If we are supporting NPM we should probably have `"dependencies"` in our `package.json` too. Like Bower, NPM can use Github as an endpoint:

```bash
npm install gaia-components/gaia-component --save
```